packages/zipkin-instrumentation-redis/src/zipkinClient.js

```diff
@@ -0,0 +1,54 @@
+const {Annotation} = require('zipkin');
+const redisCommands = require('redis-commands');
+module.exports = function zipkinClient(tracer, redis, options, serviceName = 'redis') {
+  function mkZipkinCallback(callback, id) {
+    return function zipkinCallback(...args) {
+      tracer.scoped(() => {
+        tracer.setId(id);
+        tracer.recordAnnotation(new Annotation.ClientRecv());
+      });
+      callback.apply(this, args);
+    };
+  }
+  function commonAnnotations(rpc) {
+    tracer.recordAnnotation(new Annotation.ClientSend());
+    tracer.recordServiceName(serviceName);
+    tracer.recordRpc(rpc);
+    tracer.recordBinary('redis.key', rpc);
```

Comment:
👍  should be all fixed now. Completely agree it was unnecessary. I was following the memcached example a litlte too closely (copy pasta is never a good idea).